ironfish/src/blockchain/blockchain.ts

```diff
@@ -255,16 +255,7 @@ export class Blockchain {
       defaultValue: Buffer.alloc(32),
     })
 
-    this.nullifiers = new MerkleTree({
-      hasher: new NullifierHasher(),
-      leafIndexKeyEncoding: BUFFER_ENCODING,
-      leafEncoding: new LeafEncoding(),
-      nodeEncoding: new NodeEncoding(),
-      db: this.db,
-      name: 'u',
-      depth: 32,
-      defaultValue: Buffer.alloc(32),
-    })
+    this.nullifiers = new NullifierSet({ db: this.db, storeName: 'nu' })
```

Comment:
Why does this take a store name? We'll only ever have 1 of these per blockchain DB right?

Reply:
Yea I passed the store name so that it is clear in the blockchain file which store names are currently taken. So someone else doesn't add another store with this prefix.

Reply:
If you ever try to re-use the name it'll throw an error.

Reply:
Going to leave this here for consistency since even the merkle tree takes in a name field